lib/cc/engine/analyzable_files.rb

```diff
@@ -1,39 +1,29 @@
 module CC
   module Engine
     class AnalyzableFiles
-      def initialize(directory, config)
-        @directory = directory
+      def initialize(config)
         @config = config
       end
 
       def all
         @results ||= if @config["include_paths"]
-          build_files_with_inclusions(@config["include_paths"])
+          include_paths = @config["include_paths"]
+          if include_paths == ["./"]
+            ["."]
+          else
+            filter_files(@config["include_paths"])
+          end
         else
-          build_files_with_exclusions(@config["exclude_paths"] || [])
+          raise "PLACEHOLDER: We don't support exclude_paths any longer!"
```

Comment:
I don't think that's the condition you want, we need to `raise` when we *don't* have include-paths, not when we *do* have exclude-paths.